components/azure/azureml/aml-deploy-model/src/deploymentconfig_aks.json

```diff
@@ -0,0 +1,20 @@
+{
+   "computeType": "aks",

```

Comment:
I added another file for AKS deployment. I did document it in the README.md but not sure if that shall be sufficient, @sudivate let me know if more instructions shall be added.